editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-helpers.ts

```diff
@@ -66,16 +65,20 @@ export function singleAxisAutoLayoutContainerDirections(
   container: ElementPath,
   metadata: ElementInstanceMetadataMap,
 ): SingleAxisAutolayoutContainerDirections | 'non-single-axis-autolayout' {
-  const containerElement = MetadataUtils.findElementByElementPath(metadata, container)
   const children = MetadataUtils.getOrderedChildrenParticipatingInAutoLayout(metadata, container)
-  if (containerElement == null) {
+  if (children.length === 0) {
```

Comment:
if it has 0 children we should fall back to layoutSystemForChildren

Reply:
I removed this check in https://github.com/concrete-utopia/utopia/pull/3334/commits/c19346a02a9a4afc365760a1da4b9fcdb49e1817 since `findLayoutSystemForChildren` already does the fallback you mentioned